### PR TITLE
Add openapi.json (OAS3) to published REST

### PR DESCRIPTION
**Pull request by sbont**

### CODE REVIEW

File: content/en/docs/refguide/modeling/integration/rest-services/published-rest-services/published-rest-technical-details/open-api.md [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
-tags: ["swagger", "swagger.json", "OpenAPI 2.0", "documentation", "paths", "operations", "studio pro"]
+tags: ["swagger", "swagger.json", "OpenAPI 3.0", "documentation", "paths", "operations", "studio pro"]
 ---
 
 ## 1 Introduction
 
-Every [published REST service](/refguide/published-rest-service/) is automatically documented. The system generates a *swagger.json* file that conforms to the [OpenAPI 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md) (formerly known as the "swagger specification"). This file can be [saved from Studio Pro](/refguide/published-rest-service/#export-swagger-json) or downloaded from */rest-doc/{location}/swagger.json*.
+Every [published REST service](/refguide/published-rest-service/) is automatically documented. The system generates a *openapi.json* file that conforms to the [OpenAPI 3.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.1.md). This file can be [saved from Studio Pro](/refguide/published-rest-service/#export-openapi-documentation) or downloaded from */rest-doc/{location}/openapi.json*.
```
Comment (stefanus-heath) [inline content/en/docs/refguide/modeling/integration/rest-services/published-rest-services/published-rest-technical-details/open-api.md]: here you don't explain location but in the other file you did ... does it perhaps need to be repeated here?

Comment (sbont) [reply content/en/docs/refguide/modeling/integration/rest-services/published-rest-services/published-rest-technical-details/open-api.md]: Good point... added.

**Comment by johannahemminger** [comment]
@sbont @stefanus-heath just would like to confirm the release date for this, this will be released in SP10.1 (so the July release)?

**Review by johannahemminger — APPROVED** [review]

**Comment by johannahemminger** [comment]
@MarkvanMents for SP 10.1 release in July

**Pull request merged**